onboarding/include/amb_light_service.h

```diff
@@ -15,7 +17,9 @@ typedef enum {
 
 /* USER CODE BEGIN */
 // Define light service queue config here
-
+// Settings
+static const uint8_t event_queue_len = 10;
```

Comment:
Instead of storing the parameters in a const variable, define them as a macro like we did for the light service task config above.